config/webpack.common.js

```diff
@@ -21,14 +21,14 @@ module.exports = {
   },
   module: {
     rules: [
+      // {

```

Comment:
Why is this commented?

Reply:
I wasn't sure how to proceed with eslint as there were large changes to the syntax. I can uncomment and try resolve the warnings?

Reply:
Yes please, we need to run eslint.